stable/artifactory/templates/_helpers.tpl

```diff
@@ -83,3 +83,63 @@ Scheme (http/https) based on Access TLS enabled/disabled
 {{- printf "%s" "http" -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Resolve joinKey value
+*/}}
+{{- define "artifactory.joinKey" -}}
+{{- default .Values.artifactory.joinKey .Values.global.joinKey -}}
+{{- end -}}
+
+{{/*
+Resolve joinKey value
+*/}}
+{{- define "artifactory.masterKey" -}}
+{{- default .Values.artifactory.masterKey .Values.global.masterKey -}}
+{{- end -}}
+
+
+{{/*
+Resolve joinKeySecretName value
+*/}}
+{{- define "artifactory.joinKeySecretName" -}}
+{{- default (include "artifactory.fullname" .) .Values.artifactory.joinKeySecretName .Values.global.joinKeySecretName -}}
```

Comment:
@chukka Can you explain the `include`? I don't understand its purpose here

Reply:
@danielezer  - https://github.com/jfrog/charts/blob/master/stable/artifactory/templates/artifactory-secrets.yaml#L4 - secrets from values.yaml

Reply:
Can `default` accept three arguments? This looks weird to me

Reply:
yes, it does, we are already using them - https://github.com/jfrog/charts/blob/master/stable/artifactory/templates/_helpers.tpl#L13

Reply:
Ok, cool. Good to know